# HG changeset patch
# User Rik <rik@octave.org>
# Date 1344442775 25200
#      Wed Aug 08 09:19:35 2012 -0700
# Node ID dd0e355221c612a6331d31675273a2683310c571
# Parent  4d52239daef5fb17f5afec440f717ad58342e990
octave.desktop.in: Change Octave's XDG top-level category to Education.
Suppresses warning message during install of octave.desktop and conforms
to XDG suggested hierarchy for Science & Math applications.

* octave.desktop.in: Change Octave's XDG top-level category to Education.

diff --git a/doc/icons/octave.desktop.in b/doc/icons/octave.desktop.in
--- a/doc/icons/octave.desktop.in
+++ b/doc/icons/octave.desktop.in
@@ -1,10 +1,10 @@
 [Desktop Entry]
 Name=GNU Octave
 Comment=Scientific Computing using GNU Octave
 TryExec=%OCTAVE_PREFIX%/bin/octave
 Exec=%OCTAVE_PREFIX%/bin/octave
 Icon=%OCTAVE_IMAGEDIR%/octave-logo.svg
 Terminal=true
 Type=Application
-Categories=Development;Math;Science;
+Categories=Education;Science;Math;
 StartupNotify=false
